agora_analytics/manifest.json

```
@@ -0,0 +1,55 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "a752523e-9c3d-458c-a91a-af0c9fae5adc",
+  "app_id": "agora-analytics",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "View Agora Analytics Collector metrics in Datadog",
+    "title": "Agora Analytics",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Agora Analytics Overview - Dashboard",
+        "image_url": "images/agora_analytics_dashboard.png"
+      }
+    ],
+    "classifier_tags": [
+      "Category::Monitoring",
```

I'd also recommend adding the Collaboration category since the software monitors collaboration-based technology (video streaming, calling, etc):
```suggestion
      "Category::Monitoring",
      "Category::Collaboration",
```

---

applied